start-wizard.sh

```
@@ -0,0 +1,240 @@
+restart="${restart:-Y}"
+
+if [ "$restart" != "N" ] && [ "$restart" != "n" ]; then
+    if [ ! -f "$DIR/docker-down.sh" ] || [ ! -f "$DIR/docker-up.sh" ]; then
+        printf "\n  ${RED}Error:${NC} docker-down.sh or docker-up.sh not found in %s\n\n" "$DIR"
+        exit 1
+    fi
+    printf "\n  Stopping node…\n"
+    (cd "$DIR" && bash docker-down.sh)
+    printf "\n  Starting node…\n"
+    (cd "$DIR" && bash docker-up.sh)
+    printf "\n  ${GREEN}${BOLD}Node restarted.${NC}\n\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)